tox.ini

```diff
@@ -0,0 +1,50 @@
+[tox]
+envlist =
+    quality
+    py35
+    py36
+    py37
+    clear-coverage
+    compute-coverage
+skip_missing_interpreters = True
+
+# Mark quality as a part of python3.6 build
+[travis]
+python =
+    3.6: py36, quality
+
+[testenv]
+depends = clear-coverage
+passenv =
+    CI TRAVIS TRAVIS_*
+deps =
+    codecov
+    coverage
+extras =
+    tests
+commands =
+    coverage run --parallel --source=eidas_proxy_service --branch -m unittest {posargs:discover}
```

Comment:
As a django app, this should probably use django admin to run the tests.

Reply:
Switched to django test.